proto/server/configEvtHandler_test.go

```diff
@@ -439,31 +458,260 @@ func Test_handleNetworkSlicePost_alreadyExists(t *testing.T) {
 	}
 }
 
-func Test_handleSubscriberPost(t *testing.T) {
+func Test_handleSubscriberGet5G(t *testing.T) {
+	subscriberAuthData = DatabaseSubscriberAuthenticationData{}
+	subscriber := models.AuthenticationSubscription{
+		AuthenticationManagementField: "8000",
+		AuthenticationMethod:          "5G_AKA",
+		Milenage: &models.Milenage{
+			Op: &models.Op{
+				EncryptionAlgorithm: 0,
+				EncryptionKey:       0,
+			},
+		},
+		Opc: &models.Opc{
+			EncryptionAlgorithm: 0,
+			EncryptionKey:       0,
+			OpcValue:            "8e27b6af0e692e750f32667a3b14605d",
+		},
+		PermanentKey: &models.PermanentKey{
+			EncryptionAlgorithm: 0,
+			EncryptionKey:       0,
+			PermanentKeyValue:   "8baf473f2f8fd09487cccbd7097c6862",
+		},
+		SequenceNumber: "16f3b3f70fc2",
+	}
+	subscribers := []bson.M{configmodels.ToBsonM(subscriber)}
+	subscribers[0]["ueId"] = "imsi-208930100007487"
+	dbadapter.AuthDBClient = &MockMongoSubscriberGetOne{dbadapter.AuthDBClient, subscribers[0]}
+	subscriberResult := subscriberAuthData.SubscriberAuthenticationDataGet("imsi-208930100007487")
+	if !reflect.DeepEqual(&subscriber, subscriberResult) {
+		t.Errorf("Expected subscriber %v, got %v", &subscriber, subscriberResult)
+	}
+}
+
+func Test_handleSubscriberGet4G(t *testing.T) {
+	subscriberAuthData = MemorySubscriberAuthenticationData{}
+	imsiData = make(map[string]*models.AuthenticationSubscription, 0)
```

Comment:
```suggestion
	imsiData = make(map[string]*models.AuthenticationSubscription)
```
Map does not require explicit size.